### PR TITLE
Refactor to use templates rather than fixed strings

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Also migrate to `TemplateView` to reduce code reuse and gain benefits from using a full view class.

### CODE REVIEW

File: simple_robots/views.py [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
-    if getattr(settings, ROBOTS_ALLOW_HOST_SETTING, None) == request.get_host():
-        return HttpResponse(ALLOW_ALL, content_type='text/plain')
-    return HttpResponse(DENY_ALL, content_type='text/plain')
+class ServeRobotsView(TemplateView):
+    content_type = "text/plain"
+
+    def get_template_names(self):
+        if getattr(settings, ROBOTS_ALLOW_HOST_SETTING, None) == self.request.get_host():
+            return "robots.txt"
+        return "robots-disallow.txt"
```
Comment (j4mie) [inline simple_robots/views.py]: Maybe put these in constants eg `ROBOTS_ALLOW_TEMPLATE`, `ROBOTS_DISALLOW_TEMPLATE`.

File: README.md [hunk 2]
```diff
@@ -11,18 +11,18 @@ In some cases, you do want your app to be indexed - but only in your production
...
-    
+
 Install from PIP
 
     pip install django-simple-robots
 
 In your root urlconf, add an entry as follows:
 
     from django.conf.urls import url
-    from simple_robots.views import serve_robots
```
Comment (j4mie) [inline README.md]: No need for all this API change stuff. Just put

```
serve_robots = ServeRobotsView.as_view()
```

at the end of the `views.py`.

**Review by j4mie — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (RealOrangeOne) [reply README.md]: I was tempted with that, but it felt like a bit of a hack. I think i'd rather have a tiny breaking change than this hack

Comment (j4mie) [reply README.md]: Why is it a hack? I'd really rather not have to go through all our projects and change this when we upgrade.

**Review by j4mie — APPROVED** [review]
🔥

**Comment by j4mie** [comment]
I've thought of something else this PR needs. At the moment, the README doesn't say anything about putting `simple_robots` in `INSTALLED_APPS` - because you didn't need to. Now you do.

**Review by j4mie — APPROVED** [review]
👍

**Pull request merged**